tests/functional/aliases/test_aliases.py

```diff
@@ -0,0 +1,339 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+
+models_dupe_custom_database__schema_yml = """
```

Comment:
Can we move these into a separate fixtures file? Also if there isn't a reason to preserve the naming can we follow [PEP guidance on constants](https://peps.python.org/pep-0008/#constants) (i.e. upper case and if scoped to that package preceded by an underscore)?